lib/exception_notifier/exception_track_notifier.rb

```diff
@@ -41,7 +41,7 @@ def headers_for_env(env)
       headers << "Language:    #{env['HTTP_ACCEPT_LANGUAGE']}"
       headers << "Server:      #{Socket.gethostname}"
       headers << "Process:     #{$PROCESS_ID}"
-
+      headers << "Data:        #{env["exception_notifier.exception_data"]}"
```

Comment:
What infos in `env["exception_notifier.exception_data"]`, please give me a read world example.

Reply:
for example its possible to record current user.

class ApplicationController < ActionController::Base
  before_filter :prepare_exception_notifier
  private
  def prepare_exception_notifier
    request.env["exception_notifier.exception_data"] = {
      :current_user => current_user
    }
  end
end

check this link
https://github.com/smartinez87/exception_notification/